src/main/java/org/klodnicki/entity/BookInfo.java

```diff
@@ -98,16 +93,18 @@ public String getEdition() {
     @Override
     public String toString() {
         return "BookInfo{" +
-                "title: '" + title + '\'' +
-                ", author: '" + author + '\'' +
-                ", isbn: '" + isbn + '\'' +
-                ", publisher: '" + publisher + '\'' +
-                ", publication year: " + publicationYear +
-                ", edition: '" + edition + '\'' +
-                ", genre: '" + genre + '\'' +
-                ", description: '" + description + '\'' +
-                ", language: '" + language + '\'' +
-                ", copiesNumber: " + copiesNumber +
+                "id=" + id +
+
```

Comment:
Empty line here looks like reformatting error. Should it be here?

Reply:
OK, thanks I made a correction